### ISSUE TITLE
trying to convert tensorflow fastercnn frozengraph.pb, got killed, not sure what does the debug log mean

### ISSUE THREAD

**Issue by didopop3** [openvinotoolkit/openvino#232]
`I0826 06:33:10.958902 140518601874304 shape.py:115] Do not permute data node attrs for node "predictions/Reshape" output port "0" `
`I0826 06:33:10.959511 140518601874304 shape.py:115] Do not permute data node attrs for node "predictions/Reshape/Softmax/Transpose" output port "0"`
`I0826 06:33:10.959806 140518601874304 shape.py:115] Do not permute data node attrs for node "Reshape_Transpose_Class" output port "0"`
`I0826 06:33:10.960138 140518601874304 shape.py:115] Do not permute data node attrs for node "proposals/reshape_4d" output port "0"`
`I0826 06:33:10.960672 140518601874304 shape.py:115] Do not permute data node attrs for node "reshape_proposal_4d" output port "0"`
`I0826 06:33:10.961124 140518601874304 shape.py:115] Do not permute data node attrs for node "do_reshape_conf" output port "0"`
`I0826 06:33:10.961574 140518601874304 shape.py:115] Do not permute data node attrs for node "reshape_locs_2d" output port "0"`
`I0826 06:33:10.961842 140518601874304 shape.py:115] Do not permute data node attrs for node "do_reshape_locs" output port "0"`
`I0826 06:33:10.962105 140518601874304 shape.py:115] Do not permute data node attrs for node "DetectionOutput_reshape_priors_" output port "0"`
`Killed`

The frozen graph trained with custom data got killed. (used -faster_rcnn_support_api_v1.14.json) 
The above is the debug logs.
I also tested with the original pretrained frozen graph (used /faster_rcnn_support.json) it worked successfully.

Can someone plz point to me a direction? Thanks

**Comment by lazarevevgeny** [comment]
@didopop3 is this issue still actual? Can you share the model?

**Comment by AnastasiaKazantaeva** [comment]
It seems that this issue is not actual anymore, close it. Feel free to create a new issue in cases of some questions/problems
